spec/models/tutorial_spec.rb

```diff
@@ -1,4 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe Tutorial, type: :model do
+    describe "validations" do
+        it { should validate_presence_of :title }
+        it { should validate_presence_of :description }
+        it { should validate_presence_of :thumbnail }
+    end
```

Comment:
Thank you! I also need this spec in my branch I'm working on, this (and the it exists) are helpful.